README.md

```diff
@@ -141,41 +141,37 @@ separately.
 - [Releases Page](RELEASES.md): Documentation for how to leverage our build artifacts.
 - [Roadmap for Support](ROADMAP.md): Documentation for our prioritized roadmap to support AMD GPUs.
 
-## Provisioning TheRock 🪨
+## Installing TheRock from artifacts 🪨
 
-In order to provision TheRock using either a developer/automated nightly release, a specific CI runner build or an already existing installation of TheRock, use the `build_tool/provision_machine.py` script.
+In order to install TheRock using either a developer/nightly release, a specific CI runner build or an already existing installation of TheRock, use the `build_tool/install_rocm_from_artifacts.py`. This script is used by CI to install TheRock to test machines and can be used locally to install TheRock from a specified source.
```

Comment:
Can we phrase this more like "install ROCm community builds produced by TheRock"? The specific branding/naming will help people understand that this is a build system for ROCm, not a separate project on its own.

You could also separate the CI/CD changes from user instruction updates.